### DISCUSSION TITLE
Tell me your favorite singer

### DISCUSSION THREAD

**Discussion by tsukasa1111** [NAIST-SE/AttractingContributors#1143] [2024 Attraction Contributions]
[link](https://github.com/tsukasa1111/AttractingContributors/blob/main/README.md)

This is the link to my repository. Please write it here!!!!

**Comment by Drobotsan** [comment]
Hi, I contribute to your project.
Please accept my pull request.

Would you be kind to contribute with my project :D
https://github.com/Drobotsan/Places-in-the-world

Discussion link: https://github.com/NAIST-SE/AttractingContributors/discussions/1140

**Reply by tsukasa1111** [reply]
sure!

**Comment by Tz-violet** [comment]
hi, I just suggest my favourite artist.
can you help me to complete my project: https://github.com/Tz-violet/AttractingContributors.git ?

**Reply by tsukasa1111** [reply]
sure!

**Comment by ShunHattori** [comment]
I just made a PR on your repo.
Could you contribute to my repo? Thanks in advance.
[https://github.com/ShunHattori/AttractingContributors_EssentialGadget.git](https://github.com/ShunHattori/AttractingContributors_EssentialGadget.git)
Discussion link: [#1148](https://github.com/NAIST-SE/AttractingContributors/discussions/1148)

**Comment by ryosuke-sota** [comment]
I made a PR on your repo.
Could you contribute to mine?
Here : https://github.com/ryosuke-sota/AttractingContributors.git
Thank you.

**Comment by anzhitingg** [comment]
Hello！
I have just made a PR on your repo. 
Could you contribute to mine? 
Here : https://github.com/anzhitingg/AttractingContributors-MBTI
Thank you very much！
